modules/client-configuration/pages/clients-slemicro.adoc

```diff
@@ -183,4 +200,12 @@ This window may be several hours after the client is registered.
 To speed up {micro} registration, manually reboot the client after the registration script finishes.
 ====
 
+To register your {micro} clients, you need a bootstrap repository.
```

Comment:
I copied this from the other OS.

To be honest, I think you should transform the next things into a snippet.

Reply:
I created an issue.